src/plugins/data_source_management/public/constants.ts

```diff
@@ -3,7 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { i18n } from '@osd/i18n';
 import { DirectQueryDatasourceType } from './types';
+import { DirectQueryLoadingStatus } from '../framework/types';
```

Comment:
how come the two types files but they both seem to output relatively the same level of typing. meaning it is not `DirectQueryFrameworkLoadingStatus` could it just be apart of the types files above?